scripts/real_time_ec.py

```diff
@@ -19,99 +19,11 @@
 # Local
 from crmprtd import retry
 from crmprtd.ec import makeurl, ObsProcessor, parse_xml, extract_fname_from_url
+# needs import for ec_download
```

Comment:
This doesn't really need to be said. If we import something, we assume that it's needed :)